pokemongo_bot/test/step_walker_test.py

```diff
@@ -73,13 +74,13 @@ def test_small_distance_same_spot(self):
         self.bot.config.walk_max = 1
         self.bot.config.walk_min = 1
 
-        sw = StepWalker(self.bot, 0, 0)
-        self.assertEqual(sw.dLat, 0, 'dLat should be 0')
-        self.assertEqual(sw.dLng, 0, 'dLng should be 0')
+        sw = StepWalker(self.bot, 0, 0, precision=0.0)
+        self.assertEqual(sw.dest_lat, 0, 'dest_lat should be 0')
+        self.assertEqual(sw.dest_lng, 0, 'dest_lng should be 0')
 
         self.assertTrue(sw.step(), 'step should return True')
-        self.assertTrue(self.lat == self.bot.position[0])
-        self.assertTrue(self.lng == self.bot.position[1])
+        self.assertTrue(abs(self.lat - self.bot.position[0]) < 0.01)
```

Comment:
this is 1.5 km

In [2]: haversine.haversine((0.0,0.0), (0.01, 0.01))*1000
Out[2]: 1572.533729286321

In [3]: